main.go

```diff
@@ -47,11 +47,11 @@ func main() {
 		htp.Post: handlers.CreateLease(
 			gke.NewGKEClusterLister(containerService),
 			services,
-			serverConf.Namespace,
+			serverConf.ServiceName,
```

Comment:
The above will be something we'll need to catch in functional or integration tests